tasks/calamari_setup.py

```diff
@@ -153,6 +155,24 @@ def fix_yum_repos(remote, distro):
     return True
 
 
+@contextlib.contextmanager
+def remove_epel(ctx, no_epel):
+    """
+    just remove epel.  No undo; assumed that it's used after
+    adjust_yum_repos, and relies on its state-save/restore.
+    """
+    if no_epel:
+        for remote in ctx.cluster.remotes:
+            if remote.os.name.startswith('centos'):
+                remote.run(args= [
+                    'sudo', 'rm', '-f', run.Raw('/etc/yum.repos.d/epel*')
```

Comment:
assuming yum-utils is there is why it doesn't; there's older code here and in calamari_nosetests that just manipulates the presence of the repo files.  It's brutish, but effective.